File: frontend/src/tests/PCL5Test/types/PCL5Test.ts
import TestBase from "@/schemas/TestBase";
import PCL5Criteria from "@/tests/PCL5Test/types/PCL5Criteria";
import PCL5Answer from "@/tests/PCL5Test/types/PCL5Answer";
import PCL5Question from "@/tests/PCL5Test/types/PCL5Question";

export interface PCL5Test extends TestBase {
    criterion: PCL5Criteria[];
    answers: PCL5Answer[];
    questions: PCL5Question[];
}

File: backend/app/domains/tests/pcl5/schemas/pcl5_test.py
from app.domains.tests.base.test_base import TestBase
from app.domains.tests.pcl5.schemas.pcl5_answer import PCL5Answer
from app.domains.tests.pcl5.schemas.pcl5_criteria import PCL5Criteria
from app.domains.tests.pcl5.schemas.pcl5_question import PCL5Question


class PCL5Test(TestBase):
    criterion: list[PCL5Criteria]
    answers: list[PCL5Answer]
    questions: list[PCL5Question]
